src/navigation/AppNav.tsx

```diff
@@ -19,7 +19,13 @@ export const NavApp = () => (
 
         <Route path={ROUTES.QUADRANT_PARAM} element={<QuadrantView />} />
         {/* https://gist.github.com/mjackson/b5748add2795ce7448a366ae8f8ae3bb#not-server-rendering -> should be server redirect */}
+
+        <Route path='/Digital_Technologies_Radar' element={<Radar />} />
```

Comment:
I don't thing that will work... what you can look for is the base route for react router. 

https://www.pluralsight.com/guides/add-a-base-url-to-an-app-using-redux-router-+-react-router (see section 'adding a base url')